# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gnome-color-manager/hotfix-no-man.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gnome-color-manager-3.36.2/meson.build.vanilla	2026-09-28 11:17:20.318902474 +0300
+++ gnome-color-manager-3.36.2/meson.build	2026-09-28 11:17:27.581174171 +0300
@@ -106,7 +106,7 @@
 
 subdir('data')
 subdir('help')
-subdir('man')
+#subdir('man')
 subdir('po')
 subdir('src')
 
